hasjob/static/js/app.js

```diff
@@ -184,6 +184,184 @@ window.Hasjob.PaySlider.prototype.resetSlider = function(currency) {
+  $("#job-filters-cancel").click(function(event) {
+    event.preventDefault();
+    $("#hg-sitenav").collapse("toggle");
+    $('body').removeClass('nav-open');
+  });
+
+  //On pressing ESC, close the filter dropdown if menu is open.
+  $(document).keydown(function(event) { 
+    if (event.keyCode === 27 && $("#hg-sitenav").hasClass("in")) {
+      event.preventDefault();
+      $("#hg-sitenav").collapse("toggle");
+      $('body').removeClass('nav-open');
+    }
+  });
+  
+  var scrollheight = $("#hgnav").height() - $("#hg-sitenav").height();
+  $(window).scroll(function() {
+    if($(window).width() > 767) {
+      if ($(this).scrollTop() > scrollheight) {
+        $(".header-section").slideUp();
+      }
+      else{
+        $(".header-section").slideDown();
+      }
+    }
+  });
+
   window.Hasjob.JobPost.handleGroupClick();
   $(".pstar").off().click(window.Hasjob.JobPost.handleStarClick);
+
+  var getCurrencyVal = function() {
+    return $("input[type='radio'][name='currency']:checked").val();
+  };
+
+  var setPayTextField = function(){
+    var currencyLabel = 'Pay';
+    var equityLabel = '';
+    var payFieldLabel;
+
+    if ($('#job-filters-equity').is(":checked")) {
+      equityLabel += ' + ' + '%';
+    }
+    if (getCurrencyVal().toLowerCase() === 'na'){
+      currencyLabel = 'Pay';
+    } else {
+      currencyLabel = $('#job-filters-pmin').val() + ' - ' + $('#job-filters-pmax').val();
+    }
+    if (currencyLabel === 'Pay' && equityLabel !== '') {
+      payFieldLabel = 'Equity (%)';
+    } else {
+      payFieldLabel = currencyLabel + equityLabel;
+    }
+    $('#job-filters-pay-text').html(payFieldLabel);
+  };
+
+  $('#job-filters-equity').on('change', function(){
+    setPayTextField();
+  });
+
+  // set initial value for the currency radio button
+  var presetCurrency = (Hasjob.PayFilterParameters && Hasjob.PayFilterParameters.currency) || 'NA';
+  $.each($("input[type='radio'][name='currency']"), function(index, currencyRadio){
+    if ($(currencyRadio).val() === presetCurrency) {
+      $(currencyRadio).attr('checked', 'checked');
+    }
+  });
+
+  $("input[type='radio'][name='currency']").on('change',function(){
+    setPaySliderVisibility();
+    paySlider.resetSlider(getCurrencyVal());
+    setPayTextField();
+  });
+
+  // prevent the pay filter dropdown from hiding on click
+  $('ul.pay-filter-dropdown').click(function(e) {
+    e.stopPropagation();
+  });
+
+  var setPaySliderVisibility = function(){
+    if (getCurrencyVal().toLowerCase() === 'na') {
+      $(".pay-filter-slider").slideUp();
+    } else {
+      $(".pay-filter-slider").slideDown();
+    }
+  };
+
+  var paySlider = new Hasjob.PaySlider({
+    start: (Hasjob.PayFilterParameters && Hasjob.PayFilterParameters.pmin) || 0,
+    end: (Hasjob.PayFilterParameters && Hasjob.PayFilterParameters.pmax) || 10000000,
+    selector: "#pay-slider",
+    minField: '#job-filters-pmin',
+    maxField: '#job-filters-pmax'
```

Comment:
We should standardise on one type of string quote. In Python we've standardised on double quotes for user-facing strings (meaning something that needs i18n) and single quotes for internal use string. If we carry through the same convention here, all of these should be in single quotes.